ansible/setup.sh

```
@@ -20,6 +21,14 @@ if [ -z "${POSTGRES_PASSWORD:-}" ]; then
     echo
 fi
 
+while [ -z "${ADMIN_PASSWORD:-}" ] || [[ "$ADMIN_PASSWORD" == *'$admin_password$'* ]]; do
+    if [ -n "${ADMIN_PASSWORD:-}" ]; then
+        echo 'ADMIN_PASSWORD không được chứa chuỗi $admin_password$. Vui lòng nhập lại.'
+    fi
+    read -sp "Nhập ADMIN_PASSWORD: " ADMIN_PASSWORD
```

Shellcheck (0.11.0)

[info] read without -r will mangle backslashes.

(SC2162)